frontend/todo-app/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
We currently have two package.json files. I think the `npx create-react-app` command has made a second one. We should delete the original one and create the new one made by `create-react-app`

Reply:
moved todo-app out of front-end and removed front-end folder